test/index-test.js

```diff
@@ -1,15 +0,0 @@
-/* eslint-disable no-unused-expressions */
-import {expect} from 'chai';
-import * as geoapi from '../src/index.js';
-
-describe('index', () => {
-  it('should export a geocoding interface', () => {
-    expect(geoapi.Geocoding).to.be.a('function');
-  });
-  it('should export a static maps interface', () => {
-    expect(geoapi.staticMaps).to.exist;
```

Comment:
Btw., what about avoiding the nesting and use `geoapi.staticMapsImage` and `geoapi.staticMapsUrl`?